e2e/types/types.go

```diff
@@ -11,68 +11,68 @@ import (
 // ChannelConfig defines the name and namespace for the channel CR.
 // This is not user-configurable and always uses default values.
 type ChannelConfig struct {
-	Name      string
-	Namespace string
+	Name      string `json:"name"`
+	Namespace string `json:"namespace"`
 }
 
 // NamespacesConfig are determined by distro and are not user-configurable.
 type NamespacesConfig struct {
-	RamenHubNamespace       string
-	RamenDRClusterNamespace string
-	RamenOpsNamespace       string
-	ArgocdNamespace         string
+	RamenHubNamespace       string `json:"ramenHubNamespace"`
+	RamenDRClusterNamespace string `json:"ramenDRClusterNamespace"`
+	RamenOpsNamespace       string `json:"ramenOpsNamespace"`
+	ArgocdNamespace         string `json:"argocdNamespace"`
 }
 
 // RepoConfig represents the user-configurable git repository settings.
 // It includes the repository url and branch to be used for deploying workload.
 type RepoConfig struct {
-	URL    string
-	Branch string
+	URL    string `json:"url"`
+	Branch string `json:"branch"`
 }
 
 type PVCSpecConfig struct {
-	Name                 string
-	StorageClassName     string
-	AccessModes          string
-	UnsupportedDeployers []string
+	Name                 string   `json:"name"`
+	StorageClassName     string   `json:"storageClassName"`
+	AccessModes          string   `json:"accessModes"`
+	UnsupportedDeployers []string `json:"unsupportedDeployers,omitempty"`
 }
 
 type ClusterConfig struct {
-	Kubeconfig string
+	Kubeconfig string `json:"kubeconfig"`
 }
 
 type TestConfig struct {
-	Workload string
-	Deployer string
-	PVCSpec  string
+	Workload string `json:"workload"`
+	Deployer string `json:"deployer"`
+	PVCSpec  string `json:"pvcSpec"`
 }
 
 type Config struct {
-	// User configurable values.
-	Distro     string
-	Repo       RepoConfig
-	DRPolicy   string
-	ClusterSet string
-	Clusters   map[string]ClusterConfig
-	PVCSpecs   []PVCSpecConfig
-	Tests      []TestConfig
+	// User-configurable values
```

Comment:
This change is not needed - what is wrong with "User configurable"? and removing the "." at the end is wrong. The right way is to add "." to the end of the next comment.

Reply:
Ohh yes, It was by mistake. Corrected.